scripts/ci/check-formatting.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+mkdir build -p
+pushd build
+cmake .. -DSC_CLANG_FORMAT_CODE=ON
+make clangformat_check
+popd
```

Comment:
```suggestion
cmake -B build -DSC_CLANG_FORMAT_CODE=ON
cmake --build build --target clangformat_check
```
To fix the potential issue when `build` folder already exists